### DISCUSSION TITLE
Can't access github student Developer pack

### DISCUSSION THREAD

**Discussion by kv-cache** [community/community#47038] [GitHub Education]
I'm verified as a student and I'm not able to signup for the student developer pack.

Steps I took:

1)
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/14841940/218241296-66bb2fa0-9b1d-46f2-80ba-38c66cad7cc9.png">
2)
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/14841940/218241327-50c98d4c-cde9-447b-80a6-e4c1095a261b.png">
3)
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/14841940/218241341-59e4b115-d4a5-4db4-8574-f4bcb4546207.png">

4) when I click im a student button it takes me back to step 2 and the cycle repeats

**Comment by SilasRodrigues19** [comment]
Hey @VineethKanaparthi 

I saw that you already have a Pro badge, probably the site is redirecting wrong. Try accessing this link here:

[https://education.github.com/pack/offers](https://education.github.com/pack/offers)

**Comment by lbirkert** [comment]
It looks like you already have got the PRO badge, which normally indicates that someone either bought github enterprise or they already redeemed the github student pack.

**Comment by samus-aran** [comment]
Hi @VineethKanaparthi 👋

Are you still experiencing this issue on accessing the Student Developer Pack?

**Reply by AmirulAndalib** [reply]
Hi I am having issue accessing my github student developer pack site, everything I get a 404 error although I am logged into github.

![Screenshot_20230710_020840_Chrome](https://github.com/community/community/assets/86344856/84ba5af3-895e-47f2-850b-87f0b041244f)

**Reply by andurkaronkar** [reply]
i'm also facing this issue

**Reply by j520230111** [reply]
me as well

**Reply by doopyelephant** [reply]
Me too

**Reply by shivesh-anand** [reply]
Me too

**Reply by Bellani-Wassim** [reply]
same

**Reply by andurkaronkar** [reply]
- Wait for 2-3 days it'll solved automatically

**Comment by Ashik763** [comment]
I am also facing the same problem.

**Comment by GermanMaximov** [comment]
How to get around it?
![image](https://github.com/community/community/assets/128539186/d8e18918-2ec1-4e72-affb-12ffacd89c92)

**Reply by andurkaronkar** [reply]
update your profile name as per your college documents
